README.md

```diff
@@ -1 +1,155 @@
-# my_things_ruby_capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>**Catalog of my things**</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Catalog of my things\] ](#--)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** is a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Ruby</summary>
+  <summary>IRB</summary>
+  <summary>SQL</summary>
+  <summary>Rspec</summary>
+  <summary>Vscode</summary>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Add new things]**
+- **[Schema file is created for the project]**
+- **[Data is preserve in json files]**
+- **[A console app is built for the project]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+[CLICK HERE](https://drive.google.com/file/d/1Ul1awfW1iGUGTUC3pirUVnfLqWIWhs5G/view?usp=sharing) for a video presentation of the project
```

Comment:
- [ ] Let's ensure that the video presentation is of no more than **5 minutes** duration as per the project requirements. Cheers 👍